queries/rescript/highlights.scm

```diff
@@ -66,50 +91,83 @@
 [
  (parameter (value_identifier))
  (labeled_parameter (value_identifier))
-] @parameter
+] @variable.parameter
```

Comment:
I think this was a bug before that I also had to fix when I changed the queries; it should be
```query
 (parameter (value_identifier) @parameter)
 (labeled_parameter (value_identifier) @parameter)
```
otherwise the parameter highlight is too broad.